modules/apps/scheduler/scheduler-service/service.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.3.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_3_0.dtd">
+
+<service-builder auto-namespace-tables="false" mvcc-enabled="true" package-path="com.liferay.scheduler">
+	<author>Alessio Antonio Rendina</author>
+	<namespace>Scheduler</namespace>
+	<entity local-service="true" name="SchedulerProcess" remote-service="true">
```

Comment:
This "SchedulerProcess" entity really just a definition of the scheduler trigger.
So I would call it "DispatchTrigger", to avoid the "scheduler" name.

The "SchedulerProcessLog" is capturing the execution result for the "DispatchTrigger", so I would call it "DispatchLog", or "DispatchResult".